src/js/contexts/MessageContext/MessageContext.js

```diff
@@ -39,13 +40,13 @@ export const format = (options, messages) => {
     messageObj ||
     options.defaultMessage;
 
-  const { values } = options;
+  const { labelKey, values } = options;
```

Comment:
The format function is used by a lot of components in Grommet besides just Select and SelectMultiple. These other components don't all have the concept of a `labelKey` prop. Can we explore changes that are more localized to the Select and SelectMultiple cases so that we don't have unintended changes in other components?

Reply:
Sounds reasonable, I'll see what I can do!